### PR TITLE
chore: improve code readability

### PR DESCRIPTION
**Pull request by aimuz**
Signed-off-by: aimuz <mr.imuz@gmail.com>

### All Submissions:

* [ ] Have you opened an Issue before filing this PR?
* [x] Have you signed our [CLA](https://www.mongodb.com/legal/contributor-agreement)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?
* [ ] Put `closes #XXXX` in your comment to auto-close the issue that your PR fixes (if such).

### CODE REVIEW

File: pkg/automationconfig/automation_config_builder.go [hunk 1]
```diff
@@ -217,31 +217,36 @@ func (b *Builder) setFeatureCompatibilityVersionIfUpgradeIsHappening() error {
...
-		return AutomationConfig{}, fmt.Errorf("can't build the automation config: %s", err)
-	}
-
 	dataDir := DefaultMongoDBDataDir
 	if b.dataDir != "" {
 		dataDir = b.dataDir
 	}
 
+	fcv := versions.CalculateFeatureCompatibilityVersion(b.mongodbVersion)
+	if b.fcv != "" {
```
Comment (slaskawi) [inline pkg/automationconfig/automation_config_builder.go:246]: Probably a better way is to check the `len(b.fcv)` here.

Comment (aimuz) [reply pkg/automationconfig/automation_config_builder.go:246]: Yes, it is. If you can, please help me see if there are any other problems. I will modify it once

Comment (slaskawi) [reply pkg/automationconfig/automation_config_builder.go:246]: That's the only spot I found. Otherwise, LGTM

**Comment by aimuz** [comment]
Thank you. I have made adjustments

**Comment by slaskawi** [comment]
@aimuz It seems your changes fail in our CI. Please have a look at them.

**Comment by aimuz** [comment]
Thank you. I have repaired it. Next, it should be normal and wait for the results of CI

**Review by slaskawi — APPROVED** [review]

**Comment by slaskawi** [comment]
Integrated, thanks!

**Pull request merged**